src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/JSON1006.ValueTooLarge_LocationId_Invalid.sarif

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "Sarif Functional Testing",
+          "informationUri": "https://www.example.com",
+          "version": "1.0"
+        }
+      },
+      "results": [
+        {
+          "ruleId": "TEST1001",
+          "message": {
+            "text": "Locations that specify file paths."
+          },
+          "locations": [
+            {
+              "id": 2147483648,
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "file1.cs"
+                }
+              }
+            },
+            {
+              "id": 2147483647000,
```

Comment:
>2147483647000 [](http://example.com/codeflow?start=20&length=13)

you don't need to add more than one since the one above is enough to prove ourselves that we are validating it.

Reply:
as discussed I keep this one but changed to Int 64 max.